src/doc/en/installation/conda.rst

```diff
@@ -61,38 +61,6 @@ If there are any installation failures, please report them to
 the conda-forge maintainers by opening a `GitHub Issue for
 conda-forge/sage-feedstock <https://github.com/conda-forge/sage-feedstock/issues>`_.
 
-
-.. _sec-installation-conda-source:
-
-Using conda to provide system packages for the Sage distribution
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-If Conda is installed (check by typing ``conda info``), one can install SageMath
-from source as follows:
-
-- Create a new conda environment including all standard packages
-  recognized by sage, and activate it::
-
-    $ conda env create --file environment-3.11-linux.yml --name sage-build
-    $ conda activate sage-build
-
-  If you use a different architecture, replace ``linux`` by ``macos``.
-  Alternatively, use ``environment-optional-3.11-linux.yml`` in place of
-  ``environment-3.11-linux.yml`` to create an environment with all standard and optional
-  packages recognized by sage.
-
-  A different Python version can be selected by replacing ``3.11`` by ``3.9``
-  or ``3.10`` in these commands.
-
-- Then the SageMath distribution will be built using the compilers provided by Conda
-  and using many packages installed by Conda::
-
-    $ ./bootstrap
-    $ ./configure --with-python=$CONDA_PREFIX/bin/python \
-                  --prefix=$CONDA_PREFIX
-    $ make
-
-
 .. _sec-installation-conda-develop:
```

Comment:
I am not sure about removing this section. It seems that installing sage from source within active conda environment is a supported mode. I think the files `build/pkgs/*/distros/conda.txt` are there to support this mode of installing sage from source.

That this is broken now is not a sufficient reason to drop it. If we are officially to drop it, we should do it properly (including a discussion in sage-devel and removing all files `build/pkgs/*/distros/conda.txt`).

So I suggest to update this section instead of removing it, for now.

Reply:
@saraedum - as a Sage conda maintainer, any opinion?

Reply:
> I am not sure about removing this section. It seems that installing sage from source within active conda environment is a supported mode. 

The supported mode is to install all dependencies with conda; what is not supported (in the sense that no one checks that it works or provides help if it doesn't) is to build the python dependencies using sage-the-distro, but install all non-python dependencies using conda. I also don't see any point in supporting such a mixed install.

> I think the files `build/pkgs/*/distros/conda.txt` are there to support this mode of installing sage from source.

No, these files are not used for anything. They have the same quality as the `cygwin.txt` files.

Reply:
> ... I also don't see any point in supporting such a mixed install.

I don't know... The mixed install has been in sage for years, even in broken and unsupported state. Its complete removal may need to be discussed in sage-devel if experts here have different opinions. I am not an expert. 

Let's wait for @saraedum's opinion.

> > I think the files `build/pkgs/*/distros/conda.txt` are there to support this mode of installing sage from source.
> 
> No, these files are only there to generate the conda environment files.

OK. So we should keep those files anyway. 

On the other hand, those files are also used for the mixed install. No?

Reply:
Building Sage in this way is the only way I've been able to successfully build Sage in a way that gives me a Jupyter kernel at the end of it. I think it's important that people are able to build Sage using conda in a way that gives them all Sage features (including a working Jupyter kernel). I would be against removing this unless we have another documented way of building Sage using conda such that you get a Jupyter kernel at the end of it.

Following the instructions "Using conda to provide all dependencies for the Sage library" doesn't give me a Jupyter kernel (or at least not one that is found by following [these instructions](https://doc.sagemath.org/html/en/installation/launching.html#setting-up-sagemath-as-a-jupyter-kernel-in-an-existing-jupyter-notebook-or-jupyterlab-installation)). Building from source without conda was giving me a bunch of dependency issues and I never got it to work (I'm on Fedora), which is the entire reason I use conda to build Sage.

Reply:
one way or another, we should work on a sane way to install Sage's jupyter kernel into whatever Jupiter(lab) installation is there.

The current instructions on installing it are a bit handwavey: "try something like this" etc.

Reply:
These instructions are not needed. The sagemath jupyter kernel is automatically installed in the same conda environment once you pip installed sagemath:
```bash
$ jupyter kernelspec list
Available kernels:
  python3     /opt/conda/envs/sage-dev-basic/share/jupyter/kernels/python3
  sagemath    /opt/conda/envs/sage-dev-basic/share/jupyter/kernels/sagemath
```

If you would want to install the kernel into a different env, we would need to add an install command accepting a prefix similar to https://ipython.readthedocs.io/en/latest/install/kernel_install.html#kernels-for-different-environments, or you use the path from ` jupyter kernelspec list` to install it manually in a different venv.

Reply:
> These instructions are not needed. The sagemath jupyter kernel is automatically installed in the same conda environment once you pip installed sagemath:
> 
> ```shell
> $ jupyter kernelspec list
> Available kernels:
>   python3     /opt/conda/envs/sage-dev-basic/share/jupyter/kernels/python3
>   sagemath    /opt/conda/envs/sage-dev-basic/share/jupyter/kernels/sagemath
> ```
> 
> If you would want to install the kernel into a different env, we would need to add an install command accepting a prefix similar to https://ipython.readthedocs.io/en/latest/install/kernel_install.html#kernels-for-different-environments, or you use the path from ` jupyter kernelspec list` to install it manually in a different venv.

Whether it's conda or not, one normally wants the kernel installed into the same (v)env as used for the build.
Such an installation is cheap, anyway, and can always be done, whenever  jupyter is present.

So a general feature to make it work ought to work in conda, too.

Reply:
It's actually a general feature: sage always installs the kernel specs for the same venv, see https://github.com/sagemath/sage/blob/develop/src/sage_setup/command/sage_install.py

Reply:
I could reproduce the problem with editable install. I've now added the information about the juptyer kernel in the docs.

Thanks for bringing these issues up!

Reply:
> I could reproduce the problem with editable install. I've now added the information about the juptyer kernel in the docs.
> 
> Thanks for bringing these issues up!

The explanation looks good, thanks for addressing the feedback!